freshmaker/utils.py

```diff
@@ -275,3 +278,25 @@ def is_valid_semver(version_string):
         return True
     except ValueError:
         return False
+
+@backoff.on_exception(
+    backoff.expo,
+    requests.exceptions.RequestException,
+    factor=30,
+    max_tries=3,
+    jitter=None,
+)
+def load_remote_yaml(url: str | bytes):
+    """Fetches and parses a yaml file located remotely
+    
+    :param url: Url to be requested
+    :type url: str or bytes
+    :return: The parsed yaml content
+    :rtype: Any
+    """
+
+    response = requests.get(url=url, timeout=conf.requests_timeout)
+    response.raise_for_status()
+    parsed_content = yaml.safe_load(response.content)
+
+    return parsed_content
```

Comment:
also here you could just:
`return yaml.safe_load(response.content)`

Reply:
Much better as well! Thanks a lot 😁